x86_64/assembler_test.go

```diff
@@ -48,49 +60,6 @@ func TestAssembler_Parser(t *testing.T) {
     spew.Dump(v)
 }
 
-func TestAssembler_Assemble(t *testing.T) {
```

Comment:
Moved to `x86_64/assembler_bsd_test.go` with build tag darwin to only run on Mac